ecs-cli/modules/commands/flags.go

```diff
@@ -48,6 +48,7 @@ const (
 	SubnetIdsFlag                   = "subnets"
 	VpcIdFlag                       = "vpc"
 	InstanceTypeFlag                = "instance-type"
+	InstanceRoleFlag                = "instance-role"
```

Comment:
After talking with @yinshiua, I think we're okay with leaving it as `instance-role` to keep it as clear as possible, as per @PettitWesley's comments.